scalability_and_performance/ztp_far_edge/ztp-sno-additional-worker-node.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="ztp-sno-additional-worker-node"]
+= Additional worker nodes in a {sno} cluster
```

Comment:
From a findability perspective, I think it's important to differentiate this section from https://docs.openshift.com/container-platform/4.11/nodes/nodes/nodes-sno-worker-nodes.html

```suggestion
= Adding worker nodes to {sno} clusters with GitOps ZTP
```